stix_shifter_modules/qradar/stix_translation/json/aql_events_fields.json

```diff
@@ -41,6 +41,31 @@
     "creeventlist",
     "rulename(creeventlist) as rulenames",
     "domainid",
-    "DOMAINNAME(domainid) as domainname"
+    "DOMAINNAME(domainid) as domainname",
+    "EventID",
+    "Image",
```

Comment:
by default events table in aql doesn't have any of these fields that you added. which throws invalid aql query exception.